### PR TITLE
legend intervals for cases and deaths changed, pop den units added, s…

### PR DESCRIPTION
**Pull request by lynnnakiganda**
…paces ahead of colons in popups and commas in uganda popups. Couldn't add commas to africa cases popups

### CODE REVIEW

**Comment by RonKbS** [comment]
@lynnnakiganda , this looks straight forward so I'll be merging it in. But as you go forward;

 - Keep the commit messages short.
 - You can write them as bullet points the way I am writing this.

 - Also, do not include stuff you have not done in the commit messages. `Could not add commas to Africa cases popups` for example. The commit should be about what you're adding to the code-base.

 - Correct me if I'm wrong but you are still using the atom editor and that is bringing in a lot of unnecessary changes. If you click on `Files changed` you will see spaces being added to files and nothing else of significance. This adds more work on my end when I'm reviewing the changes. Please switch to vs-code or turn off the formatter in atom.

**Comment by lynnnakiganda** [comment]
What spaces on the files?

**Comment by RonKbS** [comment]
> What spaces on the files?

These spaces @lynnnakiganda 
Annotation 2020-06-11 181040

**Comment by RonKbS** [comment]
In addition, when you're cerating new branches @lynnnakiganda ,

Please use only dashes and underscores -> `-` and `_`. This branch for example as a slash `/` which would break some terminals that tried to invoke it.

**Comment by lynnnakiganda** [comment]
okay

File: js/ug_onEachFeature_fns.js [hunk 1]
```diff
@@ -132,10 +132,10 @@ function prisons_OEF(layer) {
 
 function gdp_OEF(layer) {
     layer.bindPopup(
-        '<strong>District: </strong> ' + layer.feature.properties.DNama2017 +
```
Comment (lynnnakiganda) [inline js/ug_onEachFeature_fns.js]: these spaces i put after the colons were there on purpose

Comment (RonKbS) [reply js/ug_onEachFeature_fns.js]: I do not see the difference. On the GDP Ug map for example. Check on the map that you have on this branch and what is currently in the master branch. Share a screenshot on here if possible

**Comment by lynnnakiganda** [comment]
There's a difference because it was specifically requested by someone who finds them squeezed so they put it as a task in the change log

**Pull request merged**